core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,34 +220,29 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(
+        explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      await this.expectTitleOfExplorationSummaryTileToMatch(explorationTitle);
+    });
 
-  this.checkCompleteExplorationSection = async function(explorationTitle) {
-    var completedSectionVisible = await (
-      completedExplorationsSection.isDisplayed());
-    if (!completedSectionVisible) {
-      await this.navigateToCompletedSection();
-    }
-    await this.navigateToCompletedExplorationsSection();
+  this.navigateToCommunityLessonsAndCheckCompleteExplorations = async function(
+      explorationTitle) {
+    await this.navigateToCommunityLessonsSection();
     await this.expectTitleOfExplorationSummaryTileToMatch(
       explorationTitle);
   };
```

Comment:
This function looks identical to `navigateToCommunityLessonsAndCheckIncompleteExplorations`

Reply:
It is identical but if I write the function only once although the functionality will work, it will be ambiguous for the developer to identify which section is being checked.

Reply:
The `learnerFlow.js` test calls those two functions right after each other:

```js
    // Learner Dashboard should display 'Dummy Exploration'
    // as incomplete.
    await learnerDashboardPage
      .navigateToCommunityLessonsAndCheckIncompleteExplorations(
        'Dummy Exploration');
    // Learner Dashboard should display 'Test Exploration'
    // exploration as complete.
    await learnerDashboardPage
      .navigateToCommunityLessonsAndCheckCompleteExplorations(
        'Test Exploration');
```

If the two functions are identical, how are they checking different sections in this test?

Reply:
Done. PTAL!